app/services/experiments/table_view_service.rb

```diff
@@ -131,10 +131,10 @@ def assigned_presenter(my_module)
                             })
       end
 
-      result[:more_users_title] = user_names_with_roles(users[4..].to_a) if users.length > 3
-
+      result[:more_users_title] = users[4..].map { |u| u.full_name }.join('&#013;').html_safe if users.length > 4
```

Comment:
User's name becomes unsinitiezed?

Reply:
It's the same way as in `user_names_with_roles` method, maybe then we need also to update there?

Reply:
Yes, please